doc/_static/switcher.json

```diff
@@ -0,0 +1,105 @@
+[
+    {
+        "name": "stable/3.5.1",
```

Comment:
```suggestion
        "name": "3.5.1 (latest)",
```
I assume it looks better in the dropdown if all entries start with version numbers. I also think that "latest" is a better description than "stable".

Reply:
Well, except the directory is `stable`, so we should probably follow that?